cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -424,14 +424,22 @@
   private static final String ZOOKEEPER_CONNECT_DOC = "The zookeeper path used by the Kafka cluster.";
 
   /**
-   * <code>num.concurrent.partition.movements</code>
+   * <code>num.concurrent.partition.movements.per.broker</code>
```

Comment:
Stale name change.

Reply:
here I try to revert back name to be backwards-compatible here. and looks like the original doc has a typo.([missing `.per.broker`](https://github.com/linkedin/cruise-control/blob/a2ff74b2754f0f675208e73a14b030a9a2a081bb/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java#L427))